doc/content/getting-started/migrating-from-v2/configure-mac-settings.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Fine-tuning MAC Settings for End Devices
+weight: 60
+---
+
+{{< cli-only >}}
+
+This section is relevant for end devices that need MAC settings fine-tuning to function properly with the Network Server of {{% tts %}}.
+
+MAC settings on {{% tts %}} are configurable per end device. They can be configured from the CLI.
+
+### Setting RX1 Delay
+
+The RX1 delay of end devices is set to 1 second by default. For some end devices, this may lead to traffic being dropped. In that case, it is recommended that the RX1 delay be increased to 5 seconds:
```

Comment:
For some end devices, this may lead to downlink messages not being scheduled in time. Therefore, ...